src/api/status/public/js/build-log/build-header.js

```diff
@@ -29,6 +29,29 @@ function renderSha(compare, after) {
   buildGitSHA.innerText = after.substring(0, 7);
 }
 
+function renderLoadingAnimation(code) {
+  buildLoadingAnimation.className = 'material-icons';
+
+  /*
+  Build success return code === 0
+  Build failure  return code === 1
```

Comment:
Any non-zero result code is a failure, it won't necessarily be 1.  I think you should do:

```js
if(code === 0) { /* success */ }

if(!code) { /* no code present (i.e., it's not zero, so it's undefined/null), unknown (i.e., in progress) */ }

/* anything else is a failure */
```